tools/apigee-sackmesser/bin/sackmesser

```diff
@@ -131,5 +129,23 @@ if [[ -z "$token" && (-z "$username"  || -z "$password") ]]; then
     exit 1
 fi
 
+if [[ -z "$token" && "$apiversion" = "apigee" ]]; then
+    if [ -z "$mfa" ]; then MFA=""; else MFA="?mfa_token=$mfa"; fi
+    logdebug "Getting Apigee OAuth2 token for user $username"
+    response=$(curl -fsS -H "Content-Type:application/x-www-form-urlencoded;charset=utf-8" \
+      -H "Accept: application/json;charset=utf-8" \
+      -H "Authorization: Basic ZWRnZWNsaTplZGdlY2xpc2VjcmV0" \
+      -X POST "https://login.apigee.com/oauth/token$MFA" \
+      --data-urlencode "username=$username" \
+      --data-urlencode "password=$password" \
+      -d "grant_type=password")
```

Comment:
trivial: you mix and match long and short form arguments e.g. `--data-urlencode` and `-d`